apix_documentation_swagger_v3.yaml

```diff
@@ -168,6 +168,17 @@ info:
 
           `-H 'Accept: application/vnd.regalii.v3.2+json'`
 
+    # Pagination
+
+    For those endpoints that return a list of resources, we have added the
+    `X-Pagination` header on the response to make pagination easier. It looks
```

Comment:
`It looks something like this:` -> `Example:`

I don't like the "looks something like", maybe we should be more concise.

Reply:
What about "Following is an example of its content"?

Reply:
👍